### DISCUSSION TITLE
Can't see a debugged application using Visual Studio / Dev

### DISCUSSION THREAD

**Discussion by jose-poveda** [SafeExamBrowser/seb-win-refactoring#358] [Q&A] [answered]
Hi everyone:

I want to use SEB for my programming exams, and as such I used the following advice to setup Visual Studio and Dev until now:

https://sourceforge.net/p/seb/discussion/3704033/thread/7cc2e029/#905d

This works fine, I can open Visual Studio 2019 or Dev inside the exam without any trouble. I tried to follow further advice, as I have the same problem Frank had, not able to view any instances spawned after the original Visual Studio was launched. I created a dummy program in an easy accesible directory (C:/Prog1/Examen/Debug) taking in consideration that directory is where the .exe file is generated and whitelisted it in the SEB config section, but until now, I haven't been able to see the file working as it hides immediately. I tried with Dev, and with the release version, but I'm having no luck in displaying the screen needed to review the program.

Does anyone have any idea how to solve this issue?

**Comment by dbuechel** [comment, marked as answer]
I suggest you activate access to the application log of SEB (in the configuration tool under tab "Security"). This way, you can see in real-time what's happening behind the scenes and why the debugging instance is being hidden. I reckon it might have something to do with the original name, but that's just a guess. You can furthermore find that same information in the log files generated for each session, to be found under `%LocalAppData%\SafeExamBrowser\Logs`. Feel free to post them here in case you need help with analyzing them.

**Reply by jose-poveda** [reply]
Thank you! I checked the log, and I could pinpoint the program I needed to whitelist, so it works perfectly now.
